tests/integration/proxy/grpcproxy/register_test.go

```diff
@@ -18,7 +18,7 @@ import (
 	"testing"
 	"time"
 
-	"go.etcd.io/etcd/client/v3"
+	clientv3 "go.etcd.io/etcd/client/v3"
```

Comment:
Note this came up because I am trying to create a fork (internal to my company) that can move past `google.golang.org/grpc@v1.29.1`. In order to do this, the usage of `grpc/naming` would need to be removed, though the suggested replacement in `grpc/resolver` isn't exactly sufficient for how it gets used here.

I came across this line because in my fork the `c *clientv3.Client` on line 71 caused my editor to automatically convert this import to `go.etcd.io/etcd/clientv3` which caused a recursive reference **outside** of the repo (which in turn brought back the `grpc/naming` dependency).

In the process I noticed that the root `go.mod` [points at][1] `tests` but `tests` also **thinks** it needs to [point at][2] the root `go.mod`.

ISTM the `go.etcd.io/etcd/v3 => ../` replace line in `tests/go.mod` should be removed and potentially a replace line like `go.etcd.io/etcd/v3 => ./FORBIDDEN_DEPENDENCY` should be added? (I can do that in this PR if you like)

[1]: https://github.com/etcd-io/etcd/blob/a3174d0f8ec6ec58827d7d86448bb4df08ae69e4/go.mod#L13
[2]: https://github.com/etcd-io/etcd/blob/a3174d0f8ec6ec58827d7d86448bb4df08ae69e4/go.mod#L13